# 错误处理

监听 `error` 事件，可以捕获异步回调的错误。

```js
var http = require("http");

var opts = {
    host: "xxxoo000234sadfasdfasdf.com",
    port: 80,
    path: "/"
}

var req = http.get(opts, function(res) {
    console.log("This will never get called");
});

req.on("error", function(e) {
    console.log("Got that pesky error trapped");
});
```